synapse/lib/layer.py

```diff
@@ -78,16 +83,30 @@
 
 import synapse.lib.msgpack as s_msgpack
 
-#class LayerApi(s_cell.CellApi):
+class LayerApi(s_cell.CellApi):
 
-    #async def __anit__(self, core, link, user, layr):
+    async def __anit__(self, core, link, user, layr):
 
-        #await s_cell.CellApi.__anit__(self, core, link, user)
+        await s_cell.CellApi.__anit__(self, core, link, user)
 
-        #self.layr = layr
-        #self.liftperm = ('layer:lift', self.layr.iden)
+        self.layr = layr
+        self.liftperm = ('layer:lift', self.layr.iden)
         #self.storperm = ('layer:stor', self.layr.iden)
 
+    async def iterLayerSplices(self):
```

Comment:
I think i get the difference between the iterLayerSplices API vs the syncSplices API, but some basic doc strings would be nice :)